nuxeo-apb/apb.yml

```diff
@@ -0,0 +1,337 @@
+version: 1.0
+name: nuxeo-apb
+description: Nuxeo deployment
+bindable: False
+async: optional
+metadata:
+  displayName: nuxeo
+plans:
+  - name: nuxeo-mongo-cluster
```

Comment:
Is there a way to avoid baking the metadata back-end into the plan? It would seem ideal to defer the final deployment configuration until the binding secret can be inspected. So without a binding, an h2 provision is done. With a binding, the binding secret is inspected and the Nuxeo runtime is configured based on the specific binding secret fields. A binding secret that has a mongodb uri would cause mongodb to be configured, for example.

Reply:
Yes, I also thought about that, but at that time, I wanted to start simply by copying the template I did which is a Mongo + ES + Kafka.
It now seems I'll be able to make the template fully configurable.